### DISCUSSION TITLE
withDocument function in V3?

### DISCUSSION THREAD

**Discussion by popmatik** [sanity-io/sanity#3826] [Sanity Studio Support] [answered]
I've made use of withDocument in V2 to provide custom input components with values from the parent document. Is that function still available in V3 or is there an equivalent please? 

Thanks!

**Comment by aulonm** [comment, marked as answer]
They have a section in the migration guide to use `useFormValue` if you used `withDocument` before. https://beta.sanity.io/docs/platform/studio/v2-to-v3/migration-cheat-sheat#accessing-the-document-from-your-custom-form-component

**Reply by popmatik** [reply]
Perfect thanks so much!
